pgmpy/models/BayesianModel.py

```diff
@@ -316,20 +316,45 @@ def remove_cpds(self, *cpds):
                 cpd = self.get_cpds(cpd)
             self.cpds.remove(cpd)
 
-    def get_cardinality(self, node):
+    def get_cardinality(self, node=None):
         """
         Returns the cardinality of the node. Throws an error if the CPD for the
-        queried node hasn't been added to the network.
+        queried node hasn't been added to the network.If node is not specified returns
+        a dictionary with the given variable as keys and their respective cardinality
```

Comment:
@lohani2280 Some of the content from here can be moved to the `Returns` section. Usually try to keep this section short.

Reply:
@ankurankan Yeah , i will edit and update the pr very soon,